sympy/simplify/radsimp.py

```diff
@@ -966,6 +966,9 @@ def handle(expr):
             n *= num
             d *= num
             d = powdenest(_mexpand(d), force=symbolic)
+            if d.has(S.Zero, nan, zoo):
+                n, d = fraction(expr)
+                return _unevaluated_Mul(n, 1/d)
```

Comment:
what about `return expr`

Reply:
@sylee957 I've made the changes , any other changes to make?